src/components/Navbar.js

```diff
@@ -17,6 +17,54 @@ export function Navbar() {
   const titleInputRef = useRef(null);
   const [isLoading, setIsLoading] = useState(false);
 
+  const getAllLabels = () => {
+    const allLabels = new Set();
+    Object.values(columns).forEach(column => {
+      column.tasks?.forEach(task => {
+        task.labels?.forEach(label => {
+          allLabels.add(JSON.stringify(label));
+        });
+      });
+    });
+    return Array.from(allLabels).map(label => JSON.parse(label));
+  };
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
const getAllLabels = () => {
  const labelsMap = new Map();
  Object.values(columns).forEach(column => {
    column.tasks?.forEach(task => {
      task.labels?.forEach(label => {
        labelsMap.set(label.id, label);
      });
    });
  });
  return Array.from(labelsMap.values());
};
```
The getAllLabels function performs unnecessary JSON serialization/deserialization. Since label objects are already unique by their properties, using a Map with label.id as key would be more efficient.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;